### PR TITLE
refactor(eo): change software artifact name prefixes from `ett` to `eo`

### PR DESCRIPTION
**Pull request by LayZeeDK**
## Description
Remove the Energy Track and Trace (*ETT*) name from the Energy Origin (*EO*) product.

- Change class name prefixes from `ett` to `eo`
- Change file name prefixes from `ett` to `eo`
- Change selector prefixes from `ett` to `eo`
- Move libraries from the `ett` to the `eo` product folder
- Remove the `ett` product tag

### CI
- Overwrite Azure blobs for Development Storybook

## References
- https://github.com/Energinet-DataHub/energy-origin-issues/issues/244

### CODE REVIEW

File: libs/eo/auth/data-access-security/project.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      "options": {
+        "lintFilePatterns": [
+          "libs/eo/auth/data-access-security/src/**/*.ts",
+          "libs/eo/auth/data-access-security/src/**/*.html"
+        ]
+      },
+      "outputs": ["{options.outputFile}"]
+    }
+  },
+  "tags": ["product:eo", "domain:auth", "type:data-access"]
```
Comment (MadsThy) [inline libs/eo/auth/data-access-security/project.json:26]: Is it intentional that you name the type tag `data-access` instead of `data-access-security`? 😄

**Review by MadsThy — DISMISSED** [review]
Approved with one comment that can be ignored if my assumption is wrong.

File: libs/eo/auth/data-access-security/project.json (hunk 1 above)
Comment (LayZeeDK) [reply libs/eo/auth/data-access-security/project.json:26]: Yes, it's a `data-access` library (the library type) but it's specifically concerned with security. It's project name and path is then `<product>/<domain>/<type-prefix>-<library-name>`. We use library types to put static constraints enforced by lint rules on our inter-project dependencies as described in the readme file.

**Comment by LayZeeDK** [comment]
@MadsThy
Rebased and bumped prerelease version. Please submit another review.

**Review by MadsThy — APPROVED** [review]

**Review by MadsThy — APPROVED** [review]
I've been running off of these changes for my own branch, and with all the renames, everything still works.

**Review by MadsThy — APPROVED** [review]

**Pull request merged**